2022-06-23 10:24:13,236 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-06-23 10:24:13,247 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /home/awstclair/.local/lib/python3.9/site-packages/instabot/bot/bot.py
2022-06-23 10:24:13,256 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-06-23 10:24:13,552 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2022-06-23 10:24:13,835 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-06-23 10:24:14,126 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-06-23 10:24:14,331 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2022-06-23 10:24:16,370 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-06-23 10:24:16,371 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"The password you entered is incorrect. Please try again.","invalid_credentials":true,"error_title":"Incorrect password","buttons":[{"title":"OK","action":"dismiss"}],"status":"fail","error_type":"bad_password"}'
2022-06-23 10:24:16,371 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-06-23 10:24:16,401 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: The password you entered is incorrect. Please try again.
2022-06-23 10:24:16,415 - instabot version: 0.117.0 (api) - INFO - Error type: bad_password
2022-06-23 10:24:16,422 - instabot version: 0.117.0 (api) - ERROR - Failed to login go to instagram and change your password
2022-06-23 10:24:16,453 - instabot version: 0.117.0 (api) - INFO - Username or password is incorrect.
2022-06-23 10:26:53,652 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-06-23 10:26:53,694 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /home/awstclair/.local/lib/python3.9/site-packages/instabot/bot/bot.py
2022-06-23 10:26:53,717 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-06-23 10:26:54,011 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-06-23 10:26:54,013 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-06-23 10:26:54,014 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-06-23 10:26:54,025 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
2022-06-23 10:33:30,986 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-06-23 10:33:31,016 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /home/awstclair/.local/lib/python3.9/site-packages/instabot/bot/bot.py
2022-06-23 10:33:31,018 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-06-23 10:33:31,291 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-06-23 10:33:31,292 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-06-23 10:33:31,292 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-06-23 10:33:31,298 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
2022-06-23 10:48:30,749 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-06-23 10:48:30,767 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /home/awstclair/.local/lib/python3.9/site-packages/instabot/bot/bot.py
2022-06-23 10:48:30,772 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-06-23 10:48:31,105 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-06-23 10:48:31,112 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-06-23 10:48:31,113 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-06-23 10:48:31,137 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
